openreview/conference/templates/authorWebfield.js

```diff
@@ -301,10 +285,30 @@ var buildTableRow = function(note, reviewerIds, completedReviews, metaReview) {
     cell3.editUrl = '/forum?id=' + note.forum + '&noteId=' + metaReview.id;
   }
 
-  // Temporally hide review status
-  //return [cell0, cell1, cell2, cell3];
-  return [cell0, cell1];
-};
+  return [cell0, cell1, cell2, cell3];
+}
+
+function renderReviewSummary(data) {
+  if (!OFFICIAL_REVIEW_NAME) {
```

Comment:
Does it mean we need to set this variable when the assignments are deployed?

I would show this message if there are no reviews to show.

Reply:
I think the if is not necessary here, showing 0 reviews is good:

![Screen Shot 2020-04-08 at 4 23 46 PM](https://user-images.githubusercontent.com/545506/78830080-647a4e80-79b5-11ea-9d0a-71327a003e8e.png)

Reply:
OK, I'll remove this `if`